pyproject.toml

```diff
@@ -157,6 +157,7 @@ ignore_missing_imports = true
 [tool.pytest.ini_options]
 markers = [
     "example: marks test as example (deselect with '-m \"not example\"')",
+    "user_acceptance: marks user acceptance tests (deselect with '-m \"not user_acceptance\"')",
```

Comment:
what's the default? will running the tests from vscode run the user acceptance tests as well?

Reply:
Calling all tests from vscode on the left hand side will indeed run the user acceptance tests. Just as this will run the examples.

However, calling ``pixi run unittests`` will ignore the user acceptance tests, as well as ignore the examples, as well as run the tests in parallel. So think running all tests with ``pixi run unittests`` is adviceable, and I'd only use the VSCode test interface for subsets of unittests.